go/kbfs/libkbfs/keybase_service_base.go

```diff
@@ -1069,6 +1069,9 @@ func (k *KeybaseServiceBase) getCurrentSession(
 // CurrentSession implements the KeybaseService interface for KeybaseServiceBase.
 func (k *KeybaseServiceBase) CurrentSession(ctx context.Context, sessionID int) (
 	SessionInfo, error) {
+	ctx = CtxWithRandomIDReplayable(
```

Comment:
I don't think I understand the discrepancy between your comment about RPCs resulting from `serviceLoggedIn` and this only affecting one specific RPC.

Reply:
`CurrentSession` isn't really an RPC, it's a local call by other components to get the current session.  It can result it RPCs outside of kbfsfuse, and those are the RPCs that I want tagged.  This is the only caller of `serviceLoggedIn`, which sends is where most of those RPCs get called from.